### DISCUSSION TITLE
PCB v54 rev 1.34

### DISCUSSION THREAD

**Discussion by Karl-opec** [nliaudat/esp32_8ch_motor_shield#26] [Buy or Sell]
I can offer up to 5 Boards with 3.3 Volt Power Supply mounted. No ESP. PCB Colour red. Sending from Germany.
I have the board already here. Price is 21€ per peace plus shipping

**Comment by valig80** [comment]
I am interested on buying two boards for testing purposes. How can I buy it?

PS: shipping would be also in Germany

**Comment by Me-xx** [comment]
I am also interested in two Boards. Are they still left? Payment with PayPal? Shipping to Ingolstadt, Germany.
